config/configuration.md

```diff
@@ -201,6 +201,19 @@ The global coefficient applied to all deviation checks. Used to differentiate al
 
 Defaults to `1`.
 
+#### `asyncBeaconUpdatedeviationThresholdFactor` _(optional)_
```

Comment:
I'd maybe suggest making it required, but allow `null` to make it explicit when this is turned off.

Reply:
https://github.com/api3dao/airseeker/pull/343/commits/cafec8dfc47943d323242247581cd33f0c17321e